.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       run: |
         source $CONDA/etc/profile.d/conda.sh
         conda install conda-build
-        conda build conda.recipe --no-test
+        conda build conda.recipe

```

Comment:
we've reduce the tests in the conda recipe to basic imports so we might as well leave them on